services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    order.save()
+
+    for ticket in tickets:
+        movie_session = MovieSession.objects.get(id=ticket["movie_session"])
+        Ticket.objects.create(
+            movie_session=movie_session,
```

Comment:
it is better to set movie_session by id 
Use `movie_session_id` argument and pass the id directly.